### ISSUE TITLE
investigate RCE impact zero day Log4j

### ISSUE THREAD

**Issue by javabeanz** [javabeanz/owasp-security-logging#65]
https://www.lunasec.io/docs/blog/log4j-zero-day/

**Comment by augustd** [comment]
https://cve.mitre.org/cgi-bin/cvename.cgi?name=2021-44228

Upgraded Log4J version to 2.15.0 in #67. Working on a new release to get this into maven but having trouble with jqassistant: 

```

**Comment by javabeanz** [comment]
tried changing order of libs ? with dyn. classloading, assertions, and modules things can get hairy fast.

**Comment by augustd** [comment]
Looks like jqassistant uses the older version of Neo4J: 

```
[INFO] Plugin Resolved: jqassistant-maven-plugin-1.11.1.jar
[INFO]     Plugin Dependency Resolved: shared-1.11.1.jar
...
[INFO]     Plugin Dependency Resolved: neo4jv3-1.11.1.jar
[INFO]     Plugin Dependency Resolved: neo4j-3.5.29.jar
```

Which has some [illegal accesses](https://github.com/neo4j/neo4j/issues/11728), which (I'm guessing) threw warnings back in 2018, but won't work at all in newer versions of Java.

**Comment by javabeanz** [comment]
version 1.1.7 adresses CVE-2021-44228 - is thins one issue solved then ?

**Comment by augustd** [comment]
Yes, this is resolved in 1.1.7
